src/Lykke.AzureStorage/Tables/Decorators/LogExceptionsAzureTableStorageDecorator.cs

```diff
@@ -38,6 +40,19 @@ public LogExceptionsAzureTableStorageDecorator(INoSQLTableStorage<TEntity> impl,
             };
         }
 
+        public LogExceptionsAzureTableStorageDecorator(INoSQLTableStorage<TEntity> impl, ILogFactory logFactory)
+        {
+            _impl = impl;
+            _log = logFactory.CreateLog(this);
+
+            _dumpSettings = new JsonSerializerSettings
```

Comment:
Pay attention to the Dump(). It can throw exceptions while serialization

Reply:
Fixed